lib/galaxy/objectstore/irods.py

```diff
@@ -354,17 +374,21 @@ def _download(self, rel_path):
         except NetworkException as e:
             log.exception(e)
             return False
+        finally:
+            log.debug("irods_pt _download: %s", ipt_timer)
```

Comment:
That doesn't seem useful, that's just logging how much time `data_objects.get` takes.